packages/framework/src/Model/PriceList/PriceListFacade.php

```diff
@@ -345,4 +346,19 @@ protected function guessDelimiter(string $fileContent): string
 
         return ',';
     }
+
+    /**
+     * @param array $row
+     * @return array
+     */
+    protected function preProcessCsvRow(array $row): array
+    {
+        if (!array_key_exists(PriceListCsvColumnsEnum::PRICE, $row)) {
+            return $row;
+        }
+
+        $row[PriceListCsvColumnsEnum::PRICE] = str_replace(',', '.', $row[PriceListCsvColumnsEnum::PRICE]);
```

Comment:
Now, there is no need for the instruction "The "price" column should use a dot (.) as a decimal separator", right?

Reply:
rewritten as "recommended", better?